Packs/ProofpointEmailSecurity/Integrations/ProofpointEmailSecurityEventCollector/ProofpointEmailSecurityEventCollector.yml

```diff
@@ -0,0 +1,44 @@
+commonfields:
+  id: Proofpoint Email Security Event Collector
+  version: -1
+name: Proofpoint Email Security Event Collector
+display: Proofpoint Email Security Event Collector
+category: Analytics & SIEM
+description: Collects events for Proofpoint using the streaming API.
+configuration:
+- display: Server Host
+  name: host
+  type: 0
+  defaultvalue: logstream.proofpoint.com:443
+  required: true
+- display: Cluster ID
+  name: cluster_id
+  type: 0
+  required: true
+- displaypassword: API key
+  name: api_key
+  hiddenusername: true
+  type: 9
+  required: true
+- display: Fetch interval in seconds
+  name: fetch_interval
+  type: 0
+  defaultvalue: 60
+  required: true
+- display: Use system proxy settings
+  name: proxy
+  type: 8
+  required: false
+script:
+  script: ""
+  type: python
+  commands:
+    []
+  dockerimage: demisto/websockets:11.0.3.71670
+  longRunning: true
+  subtype: python3
+marketplaces:
+- marketplacev2
+fromversion: 6.8.0
```

Comment:
not sure this makes sense for xsaim.

Reply:
That's the value we have for the current collectors. I think it means that it will work on any version of XSIAM.